modules/src/main/java/org/jboss/additional/testsuite/jdkall/present/core/jca/DatasourcesActiveCountTestCase.java

```diff
@@ -53,7 +53,7 @@
  */
 @RunWith(Arquillian.class)
 @RunAsClient
-@EapAdditionalTestsuite({"modules/testcases/jdkAll/Eap7/management/src/main/java","modules/testcases/jdkAll/Eap71x-Proposed/management/src/main/java","modules/testcases/jdkAll/Eap71x/management/src/main/java","modules/testcases/jdkAll/Wildfly/management/src/main/java"})
+@EapAdditionalTestsuite({"modules/testcases/jdkAll/Eap71x-Proposed/management/src/main/java#7.1.4","modules/testcases/jdkAll/Eap71x/management/src/main/java#7.1.4","modules/testcases/jdkAll/Wildfly/management/src/main/java#14.0.0"})
```

Comment:
Could we use also the comment approach for now, as we are not sure when this will be included (after all it is just an estimation)?

Reply:
No problem, done